src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs

```diff
@@ -10789,6 +11284,45 @@ static int M(out int x)
             Assert.False(verifier.HasLocalsInit("C..ctor"));
         }
 
+        [Fact]
+        public void SkipLocalsInitFieldInitializer()
```

Comment:
>SkipLocalsInitFieldInitializer [](start = 20, length = 30)

It doesn't look like this test actually tests anything in field initializer. However, I think this is an interesting scenario to test. We could have a lambda with a local function within a field initializer. We also could have two constructors: one with an attribute and one without. It feels like the attributes on the constructors shouldn't have any effect on the lambda and the local function. But the attribute on the type/module should have an effect. Please add a test like this.

Reply:
What's a more appropriate term for this test? "ImplicitConstructor"?

---
In reply to: [371973916](https://github.com/dotnet/roslyn/pull/41183#discussion_r371973916) [](ancestors = 371973916)

Reply:
> What's a more appropriate term for this test? "ImplicitConstructor"?

Yes, I think this term will describe the purpose of the test more accurately.

---
In reply to: [371989150](https://github.com/dotnet/roslyn/pull/41183#discussion_r371989150) [](ancestors = 371989150,371973916)